### PR TITLE
New Blog for Monday.

### PR DESCRIPTION
**Pull request by robvsmith**
### Fixes #

### CODE REVIEW

File: content/news/194-the-calm.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+I mention this as a way of saying thanks. It is that time of year after all.
+
+At the root of this whole thing is the idea of empowering the individual.
+
+Without everyone in the community this little blockchain marketplace project would be nothing. It is your project as much as it is anyone's else, and that is just the way we want it.
+
+[Be sure to follow us on Twitter](https://twitter.com/lbryio), as the 12 Days Of Devmas are going on. We're giving out LBC everyday to celebrate! You'll want to have participated come next year, as there are going to be more ways to use LBC on the LBRY network than ever before.
+
+###A New Year
```
Comment (tzarebczan) [inline content/news/194-the-calm.md]: should this be ## like Gratitude?

File: content/news/194-the-calm.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Thanks for the great summary, and the memes!
+
+##Gratitude
+
+I mention this as a way of saying thanks. It is that time of year after all.
+
+At the root of this whole thing is the idea of empowering the individual.
+
+Without everyone in the community this little blockchain marketplace project would be nothing. It is your project as much as it is anyone's else, and that is just the way we want it.
```
Comment (tzarebczan) [inline content/news/194-the-calm.md]: comma after community?

File: content/news/194-the-calm.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+author: rob-smith
+title: 'A Thankful New Year'
+date: '2018-12-17 8:00:00'
+cover: 'bc-calm.jpg'
+---
+#Community
+
+It has been a big year for the LBRY team.
+
+To be clear when I use the word "team" I am referencing the community along with the company. Whether it is retweeting or commenting on Twitter, sharing a new meme with Julie and Tom, or uploading content onto the network, you know that this organization is defined by its community.
```
Comment (tzarebczan) [inline content/news/194-the-calm.md]: first sentence missing a comma ? after clear or team?

File: content/news/194-the-calm.md (hunk 1 above)
Comment (robvsmith) [reply content/news/194-the-calm.md]: Nah, I did three on purpose.

**Pull request merged**